### ISSUE TITLE
twisted.test.test_ftp.FTPServerPasvDataConnectionTestCase. deletes. useless

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3321]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#3321|
|Type|defect|
|Created|2008-06-28 16:26:57Z|
|Branch|https://github.com/twisted/twisted/tree/test-ftp-3321|

windows. bad.

Attachments:

* [ftp.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a45/a45a8131ead612928818b0a1908f0989c1197749/0de399c06f87caeab252614aa73c0f7c0ff3108d.diff) (619 bytes) - added by therve on 2008-06-28 16:28:11Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3321 3321
type__defect defect
reporter__therve therve
priority__low low
milestone__ 
branch__branches_test_ftp_3321 branches/test-ftp-3321
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ftp ftp
keywords__ 
time__1214670417000000 1214670417000000
changetime__1214729994000000 1214729994000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @therve|
|-|-|

I wonder if the rest of that `ebTearDown` (misnamed method, btw, since it's a callback) is important.  Maybe we can delete it, and the `tearDown` to, in favor of some `addCleanup` calls elsewhere?

Also, please update copyright statement date.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve commented|
|-|-|

(In [24155]) Branching to 'test-ftp-3321'

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set status to `closed`|
|-|-|

(In [24159]) Merge test-ftp-3321

Author: therve
Reviewer: exarkun
Fixes [#3321](https://github.com/twisted/twisted/issues/3321)

Remove useless shutil.rmtree done in FTP server tests, and replace some
tearDown methods by addCleanup calls.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
